projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -206,65 +206,63 @@
 
    public static void applyBaseDir(
          TestrigSettings settings, Path containerDir,
-         String testrig, String envName, String questionName) {
```

Comment:
I think this is leftover from before the last time we restructured testrig layout